org.gimp.GIMP.Plugin.BIMP.metainfo.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.gimp.GIMP.Plugin.BIMP</id>
+  <extends>org.gimp.GIMP.desktop</extends>
```

Comment:
The correct id is `org.gimp.GIMP` https://gitlab.gnome.org/GNOME/gimp/-/blob/master/desktop/org.gimp.GIMP.appdata.xml.in.in#L4
```suggestion
  <extends>org.gimp.GIMP</extends>
```

Reply:
Same comment on other extensions, otherwise lgtm

Reply:
to be fair I forgot where I picked up the .desktop but it's everywhere :-/ Will fix.

well actually this is where I picked it up:

https://github.com/flathub/io.github.Hexchat.Plugin.OTR/blob/7ec0c67ae98d186982db94b29cb9b2f58b866a6a/io.github.Hexchat.Plugin.OTR.metainfo.xml#L4

Reply:
done